### PR TITLE
Update set-up-node-with-installer.md

### PR DESCRIPTION
**Pull request by dubrado**
Just some small edits to the language and format in some areas to clean things up for the newer node builders with less experience.

### CODE REVIEW

File: docs/nodes/build/set-up-node-with-installer.md
Comment (yulin-dong) [inline docs/nodes/build/set-up-node-with-installer.md:24]: Should docusaurus admonition https://docusaurus.io/docs/markdown-features/admonitions be used here?

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: docs/nodes/build/set-up-node-with-installer.md
Comment (yulin-dong) [inline docs/nodes/build/set-up-node-with-installer.md:183]: admonition

**Review by yulin-dong — CHANGES_REQUESTED** [review]

**Comment by cgcardona** [comment]
Please merge the latest from `master` into your PR.

**Review by cgcardona — DISMISSED** [review]
LGTM once you merge `master` into your branch and after the requested admonitions are in place.

File: docs/nodes/build/set-up-node-with-installer.md
Comment (yulin-dong) [inline docs/nodes/build/set-up-node-with-installer.md:24]:
```suggestion
::: note
 HW requirements shall scale with the amount of AVAX staked on
```

add the closing :::

**Review by yulin-dong — CHANGES_REQUESTED** [review]

**Review by yulin-dong — APPROVED** [review]

**Pull request merged**
